ouroboros-network-framework/src/Ouroboros/Network/IOSim.hs

```diff
@@ -97,9 +97,21 @@ data Connection m = Connection
       -- | SDU size of a connection.
       --
     , connSDUSize       :: !SDUSize
+
+      -- | Time until when this connection could be simultnaneously openned.  If
```

Comment:
openned -> opened

Reply:
I need to use `:set spell` more often!

Reply:
simultnaneously -> simultaneously

This one is super tricky to get right to me 😄